seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/FileUtils.java

```diff
@@ -105,20 +105,20 @@ public static Long getFileLineNumber(@NonNull String filePath) {
      */
     public static Long getFileLineNumberFromDir(@NonNull String dirPath) {
         File file = new File(dirPath);
-        Long value = null;
         if (file.isDirectory()) {
-            value = Arrays.stream(file.listFiles()).map(currFile -> {
+            File[] files = file.listFiles();
+            if (files == null) {
+                return 0L;
+            }
+            return Arrays.stream(file.listFiles()).map(currFile -> {
```

Comment:
Here `file.listFiles()` suggests to use the above variable `files`.

Reply:
+1